addOns/pscanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -161,6 +161,10 @@ <H2>Strict Transport Security Header Scanner</H2>
 
 Redirects to HTTPS URLs on the same domain will only be reported at Low threshold.
 
+<H2>Sub Resource Integrity attribute missing</H2>
+This scanner checks if the integrity attribute in script or link element served by a CDN.<br>
```

Comment:
Is missing? Exists? This sentence isn’t quite right.

Reply:
It should also not say `by a CDN`, the scanner is not checking from what/where it's served just that it's external (the description of the alert looks better).